lib/components/Badge/badge.css

```diff
@@ -0,0 +1,78 @@
+.badge-component {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+}
+
+.ava-round-img {
+    border-radius: 50%;
+}
+
+.badge-dot {
```

Comment:
You can make your badge look more distinct by adding white border to it.

Reply:
Thanks for the suggestion, has been implemented.